### DISCUSSION TITLE
Does moby install docker bin on WSL?

### DISCUSSION THREAD

**Discussion by eirisdg** [rancher-sandbox/rancher-desktop#1473] [Q&A] [answered]
Hi.
I have a doubt. 
When Rancher Desktop is installed in Windows with DockerD (Moby), in Powershell Docker command runs well, but in WSL Docker command is not found.
Is docker command included in the integration with wsl?

By the way, i've installed it with brew, but doesn't work at all, it doesnt show output logs.
For example:
![image](https://user-images.githubusercontent.com/5367659/152323659-5c0f82a1-1466-4a0a-93e4-30d34eebd219.png)
When in Powershell:
![image](https://user-images.githubusercontent.com/5367659/152323740-fd68df38-2dd3-42b5-8940-fa2d28e55a66.png)

**Comment by eirisdg** [comment, marked as answer]
Ok, according to this discussion (https://github.com/rancher-sandbox/rancher-desktop/discussions/1215) I can answer myself.

WSL integration doesn't install docker-ce-cli. So to install it:
```
# curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
# sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
# sudo apt install docker-ce-cli
```

BTW, the result is the same:
![image](https://user-images.githubusercontent.com/5367659/152336589-278be7e0-5b22-4d4c-9bc6-e5a015926cf9.png)

So we can use brew binary or docker binary, that in WSL we dont have output.

**Comment by mook-as** [comment]
Additionally, we _do_ ship the Linux docker clients; however, if you have disabled [`appendWindowsPath`](https://docs.microsoft.com/en-us/windows/wsl/wsl-config#interop-settings) in `/etc/wsl.conf` then it wouldn't be able to find it.  We do not attempt to write them into your WSL distribution.

**Reply by eirisdg** [reply]
Hmmm. I think this point should be asked during installation, or at least shown as info or warning. Thanks for your answer.
